integration_tests/tests/assess/view_application_overview.cy.ts

```diff
@@ -9,11 +9,35 @@
 //    When I visit the application overview for that application
 //    Then I see the submitted application overview
 
-import { submittedApplicationFactory } from '../../../server/testutils/factories/index'
+import { faker } from '@faker-js/faker'
+import {
+  externalUserFactory,
+  statusUpdateFactory,
+  submittedApplicationFactory,
+} from '../../../server/testutils/factories/index'
+import { DateFormats } from '../../../server/utils/dateUtils'
 import SubmittedApplicationOverviewPage from '../../pages/assess/submittedApplicationOverviewPage'
 
 context('Assessor views a submitted application overview', () => {
-  const submittedApplication = submittedApplicationFactory.build()
+  const submittedApplication = submittedApplicationFactory.build({
+    statusUpdates: [
+      statusUpdateFactory.build({
+        updatedAt: DateFormats.dateObjToIsoDateTime(faker.date.past()),
+        updatedBy: externalUserFactory.build({ username: 'Anne Assessor' }),
```

Comment:
would it be clearer here if the fixture had a realistic *username* like `"ANNE_ASSESSOR"` to make it obvious that what's currently available and used is a username and not a human name?